examples/components/toast/toast.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import router from 'hoc-little-router'
+import classnames from 'classnames'
+import { onlyUpdateForPropTypes } from 'recompose'
+import styles from './toast.styles.scss'
+
+const ToastExample = ({ style, className, onClick }) => {
+  const classes = classnames(
+    styles.example,
+    className,
+  )
+
+  return (
+    <div style={style} className={classes} >
+      <div className={styles.button} onClick={() => onClick({ title: 'Default' })}>default</div>
```

Comment:
Everytime you rerender, you are instanciating a new function, so the shallowEquality is `false` and the sub render (children) will not be optimized.

I know this is an example site, but this is a good practice to have and something to know :)